editor.planx.uk/src/@planx/components/shared/Preview/SummaryList.tsx

```diff
@@ -331,19 +332,13 @@ function SummaryList(props: SummaryListProps) {
         confirmText="Yes"
         cancelText="No"
       >
-        <Typography>
-          Changing this answer means you will need to confirm any other answers
-          after it. This is because:
-          <ul>
-            <li>
-              a different answer might mean the service asks new questions
-            </li>
-            <li>
-              your planning officer needs the right information to assess your
-              application
-            </li>
-          </ul>
-        </Typography>
+        <ReactMarkdownOrHtml
+          // sketchy way to find the Review node in the flow to get the disclaimer, breaks if more than one review node in flow
+          source={
+            Object.values(props.flow).find((node) => node.type === TYPES.Review)
+              ?.data?.disclaimer
+          }
```

Comment:
Aha welcome to React land :dizzy: this is where the concept of "props" come in ! https://react.dev/learn/passing-props-to-a-component

So now, the top-level public/preview `Component` in Review/Public/index.tsx` has access to `props.disclaimer` which you've defined in the editor & model.ts and, instead of `find`, we can pass this down the component tree to `SummaryList` (going through a `Presentational` component in the middle) ! 

So try something like :
1. `<Presentational disclaimer={props.disclaimer} ... />`
2. `<SummaryListsBySection disclaimer={props.disclaimer} ... />`
3. Then here `source={props.disclaimer}` :slightly_smiling_face: